### DISCUSSION TITLE
群侠传启动 20230510发布

### DISCUSSION THREAD

**Discussion by chenggongchina** [jynew/jynew#1487] [Announcements]
同时包含三端（steam、taptap、AppStore）的更新

MOD开发者须以此版本进行打包（release分枝：[https://github.com/jynew/jynew/tree/release](https://github.com/jynew/jynew/tree/release)）

主要修复了各端无法使用lua hotfix的问题，并且进行了实机单元测试

<hr /><em>This discussion was created from the release <a href='https://github.com/jynew/jynew/releases/tag/20230510'>群侠传启动 20230510发布</a>.</em>

**Comment by kaclok** [comment]
顶
